yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -399,7 +412,8 @@ function makeL2BlockProposedEvent(l1BlockNum: bigint, l2BlockNum: bigint, archiv
  * @param l2BlockNumber - The L2 block number of in which the message was included.
  * @returns MessageSent event logs.
  */
-function makeMessageSentEvent(l1BlockNum: bigint, l2BlockNumber: bigint, index: bigint) {
+function makeMessageSentEventWithIndexInSubtree(l1BlockNum: bigint, l2BlockNumber: bigint, indexInSubtree: bigint) {
```

Comment:
here I would nuke l2BlockNumber from arg and just have the global index on the input.